validator/slashing-protection/local/standard-protection-format/import.go

```diff
@@ -9,7 +9,7 @@ import (
 	"io/ioutil"
 
 	"github.com/pkg/errors"
-	"github.com/prysmaticlabs/eth2-types"
+	types "github.com/prysmaticlabs/eth2-types"
```

Comment:
I suppose `go imports` doesn't know that `types` is redundant here -- and it is redundant (code will compile and run w/o it - so, following the Occam's razor I didn't use it). Let's remove it?

Reply:
I err on the side of leaving the redundant alias. We should use `go imports` for cleanup and it will be tedious to remember that a certain alias is not necessary. Other than redundancy, there is no drawback of including the alias. right?

Reply:
I also agree on leaving the alias, otherwise we'll have a lot of code review that has comments about the same thing in the future. It's ok to embrace this default, IMO